src/utils/asyncStorage.js

```diff
@@ -0,0 +1,47 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export async function getMyStringValue (key) {
+  try {
+      return await AsyncStorage.getItem(key)
+  } catch(e) {
+      console.log(e)
+  }
+}
+
+export async function setStringValue (key, value) {
+  const stringValue = JSON.stringify(value)
+  try {
+    await AsyncStorage.setItem(key, stringValue)
+  } catch(e) {
+    console.log(e)
+  }
+}
+
+export async function getAllKeys () {
+  let keys = []
+  try {
+    keys = await AsyncStorage.getAllKeys()
+    console.log(keys)
+    return await AsyncStorage.getAllKeys()
```

Comment:
do you want to just return all keys here? you are calling `getAllKeys` twice

Reply:
I want to keep it similar to the format of the other functions, returning a promise. Ill remove the log

Reply:
you mean remove the awaits?